src/index.scss

```diff
@@ -1,4 +1,7 @@
 @import './variables.scss';
+@import './components/Header/Header.scss';
```

Comment:
I'd recommend to use scss modules instead of importing each styles file into index.scss. The more such imports you have the more chances for styling conflicts you have. Could you please fix?